src/api/app/views/webui/user/show.html.haml

```diff
@@ -0,0 +1,170 @@
+- @pagetitle = "Home of #{@displayed_user}"
+- @layouttype = 'custom'
+- @crumb_list = [@pagetitle]
+/ user
+#userinfo.grid_4.alpha.box.box-shadow
+  / avatar
+  = user_icon(@displayed_user, 200, 'home-avatar') if @displayed_user.email
+  / info
+  %h1
+    %span#home-realname
+      - if @displayed_user.realname.present?
```

Comment:
why is this `if` needed? if it is not present we will render an empty string or a nil. So it won't have any effect, right?